src/checklist/ChecklistMultiSignOrVerify.tsx

```diff
@@ -0,0 +1,169 @@
+import { Button, Checkbox } from '@equinor/eds-core-react';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { AsyncStatus, ItemToMultiSignOrVerify } from '../services/apiTypes';
+import { ProcosysApiService } from '../services/procosysApi';
+import { COLORS } from '../style/GlobalStyles';
+
+const Wrapper = styled.div`
+    position: fixed;
+    top: 0;
+    bottom: 66px;
+    left: 0;
+    width: 100vw;
+    background-color: ${COLORS.white};
+    z-index: 100;
+    padding: 24px;
+    overflow: auto;
+    box-sizing: border-box;
+`;
+
+const ButtonWrapper = styled.div`
+    display: flex;
+    margin-top: 12px;
+    & > :first-child {
+        margin-right: 12px;
+    }
+`;
+
+const OptionsWrapper = styled.div`
+    background-color: ${COLORS.fadedBlue};
+`;
+
+type ChecklistMultiSignOrVerifyProps = {
```

Comment:
setIsSigned and setIsVerified are not used in the ChecklistMultiSignOrVerify component. Is there a reason why they are in its prop type?
+ If you keep the same order of the props in both the prop type and in the component args it's easier to read